### PR TITLE
Updates for Moto/Boto

### PR DESCRIPTION
**Pull request by jordanvance**

### CODE REVIEW

File: src/shoobx/mocks3/models.py [hunk 1]
```diff
@@ -182,6 +179,15 @@ def etag(self):
                 self._etag = file_hash.hexdigest()
         return f'"{self._etag}"'
 
+    @property
+    def checksum_value(self):
+        return self._etag
+
+    @checksum_value.setter
+    def checksum_value(self, value):
+        self.etag
```
Comment (agroszer) [inline src/shoobx/mocks3/models.py:188]: huh, maybe self.etag=value and not pass?

File: src/shoobx/mocks3/tests/test_config.py [hunk 2]
```diff
@@ -5,11 +5,11 @@
 ###############################################################################
 """Shoobx S3 Config Test
 """
-import mock
 import os
 import shutil
 import tempfile
 import unittest
+from unittest import mock
```
Comment (agroszer) [inline src/shoobx/mocks3/tests/test_config.py:12]: can't see mock usage added in the diff, pyflakes does not complain?

Comment (jordanvance) [reply src/shoobx/mocks3/tests/test_config.py:12]: this is pyupgrade changing from `import mock` to `from unittest import mock`

File: src/shoobx/mocks3/models.py (hunk 1 above)
Comment (jordanvance) [reply src/shoobx/mocks3/models.py:188]: why use the value? `self.etag` already creates an md5 hash.

Comment (jordanvance) [reply src/shoobx/mocks3/models.py:189]: done

**Review by agroszer — APPROVED** [review]

**Pull request merged**